IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormCommitTests.cs

```diff
@@ -244,7 +246,7 @@ public void Should_stage_only_filtered_on_StageAll()
 
                 testform.StageAllToolItem.PerformClick();
 
-                var fileNotMatchedByFilterIsStillUnstaged = testform.UnstagedList.AllItems.Where(i => i.Item.Name == "file2.txt").Any();
+                var fileNotMatchedByFilterIsStillUnstaged = testform.UnstagedList.AllItems.Any(i => i.Item.Name == "file2.txt");
```

Comment:
Is this required for this PR or is it fixing any other test failures in FormCommit?
(I occasionally see them locally)

Reply:
No, I've changed this out of habit for better readability. If that's unacceptable and shouldn't be here as it's unrelated, then I can revert it.

Reply:
It is better, keep